MODELS/comments.models.js

```diff
@@ -0,0 +1,12 @@
+const db = require('../db/connection');
```

Comment:
Be careful when making commits that you only add the necessary files, this file is not needed for this commit!